app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list[dict]) -> list[Person]:
+
+    for person_dict in people_list:
+        Person(person_dict["name"], person_dict["age"])
```

Comment:
This line creates a `Person` instance for each dictionary in `people_list`. Ensure that each dictionary contains the keys 'name' and 'age' to avoid KeyError.